portfolio/src/main/webapp/script.js

```diff
@@ -18,6 +18,8 @@
 function getComments() {
   const commElem = document.getElementById('comments-container');
   commElem.innerHTML = "";
+  var buttonDiv = document.getElementById("delete");
```

Comment:
Is there a specific reason you decided to move the delete button here? I was thinking that you had the delete button inside the `if (comments.length > 0)` block so that you would only be showing the delete button when there are a non-zero number of comments.

Reply:
I moved it up here so that I can empty the button div every time I get comments. I only want to show the delete button if comments are displayed. 
I ran into the issue where if a user asked to see say 5 comments, but then changed that number to 0, the delete button would still show (even though no comments were showing). I realized this was because I was defining and clearing the button div within the conditional. That's why I moved it up, but if there is a better solution I would be happy to adapt.

Reply:
Got it, that sounds good; thanks for the clarification!